devito/propagator.py

```diff
@@ -244,7 +247,7 @@ def generate_loops(self, loop_body):
             time_stepping = self.get_time_stepping()
         else:
             time_stepping = []
-        loop_body = omp_for + [loop_body] if self.compiler.openmp else [loop_body]
+        loop_body = [cgen.Block(omp_for + loop_body)]
```

Comment:
It solves a bug. Where openmp pragma was written in form like this:
#pragma openmp for
{
for(...)

which caused compiling error